app/app_stack.py

```diff
@@ -0,0 +1,38 @@
+from aws_cdk import (
+    Duration,
+    Stack,
+    aws_lambda as _lambda,
+    aws_events as events,
+    aws_events_targets as targets,
+    aws_s3 as s3,
+)
+from constructs import Construct
+
+
+class OdinSolarStack(Stack):
+    def __init__(self, scope: Construct, id: str, **kwargs) -> None:
+        super().__init__(scope, id, **kwargs)
+
+        solar_bucket = s3.Bucket.from_bucket_name(
+            self, "SolarBucket", "odin-solar"
+        )
+
+        download_solar = _lambda.DockerImageFunction(
+            self,
+            "OdinSolarDownloadFunction",
+            code=_lambda.DockerImageCode.from_image_asset("./app/handler"),
+            architecture=_lambda.Architecture.X86_64,
+            timeout=Duration.seconds(60),
+            memory_size=256,
+        )
```

Comment:
Behöver denna verkligen vara en docker-function? Hade inte en mer lättviktig python-runtime räckt?

Reply:
max 250Mb är gränsen requirements.txt drar in 360 Mb.